nerdlets/e2m-gui-nerdlet/account-details.js

```diff
@@ -1,110 +1,109 @@
-							text={""}
-						/>
-					)}
-				</div>
-			</div>
-		);
-	}
-	render() {
-		if (!this.props.cardinalities || !this.props.cardinalities.length) {
-			return this.getProgressDiv();
-		}
+                // Colors
+                pathColor: '#469990',
+                textColor: '#f88',
+                trailColor: '#d6d6d6',
+                backgroundColor: '#3e98c7'
+              })}
+              value={this.props.percentCompleteCardLoading}
+              text=""
+            />
+          )}
+        </div>
+      </div>
+    );
+  }
 
-		const {
-			timerangeArray,
-			cardinalities,
-			cardinalityTotals,
-			cardinalityDataLoading
-		} = this.props;
+  render() {
+    if (!this.props.cardinalities || !this.props.cardinalities.length) {
+      return this.getProgressDiv();
+    }
 
-		const dataByRule = buildCardinalityChartData(
-			timerangeArray,
-			cardinalities
-		);
+    const {
+      timerangeArray,
+      cardinalities,
+      cardinalityTotals,
+      cardinalityDataLoading
+    } = this.props;
 
-		const dataByAccount = buildCardinalityTotalsChartData(
-			timerangeArray,
-			cardinalityTotals
-		);
+    const dataByRule = buildCardinalityChartData(timerangeArray, cardinalities);
 
-		return (
-			<div className="CardinalityDetailsPaneInner">
-				{!dataByRule || !dataByRule.length ? null : (
-					<div className="CardinalityDetailsChartWrapper">
-						<h3>Limits Per Rule</h3>
-						<LineChart data={dataByRule} fullWidth fullHeight />
-					</div>
-				)}
-				{!dataByAccount ||
-				!dataByAccount.length ||
-				cardinalityDataLoading ? (
-					this.getProgressDiv()
-				) : (
-					<div className="CardinalityDetailsChartWrapper">
-						<h3>Limits per Account</h3>
-						<LineChart data={dataByAccount} fullWidth fullHeight />
-					</div>
-				)}
-				<div className="refreshcache">You are seeing cached data. To refresh click <span onClick={() => this.props.refreshCardinality()}>here</span>.</div>
-			</div>
-		);
-	}
+    const dataByAccount = buildCardinalityTotalsChartData(
+      timerangeArray,
+      cardinalityTotals
+    );
+
+    return (
+      <div className="CardinalityDetailsPaneInner">
+        {!dataByRule || !dataByRule.length ? null : (
+          <div className="CardinalityDetailsChartWrapper">
+            <h3>Limits Per Rule</h3>
+            <LineChart data={dataByRule} fullWidth fullHeight />
+          </div>
+        )}
+        {!dataByAccount || !dataByAccount.length || cardinalityDataLoading ? (
+          this.getProgressDiv()
+        ) : (
+          <div className="CardinalityDetailsChartWrapper">
+            <h3>Limits per Account</h3>
+            <LineChart data={dataByAccount} fullWidth fullHeight />
+          </div>
+        )}
+        <div className="refreshcache">
+          You are seeing cached data. To refresh click{' '}
+          <span onClick={() => this.props.refreshCardinality()}>here</span>.
+        </div>
+      </div>
+    );
+  }
 }
 
```

Comment:
Diff lost sync, check files.

Reply:
Linting rules adjusted the format via `npm run eslint-fix`